test_transfluent.py

```diff
@@ -19,17 +19,23 @@ def make_response(content, status_code=200):
     response = requests.Response()
     response.status_code = status_code
     response._content = content
+    response.encoding = 'utf-8'
     return response
 
 
 def make_error_response():
+    content = (
+        '{"status":"ERROR","error":{"type":"EBackendParameterInvalid","mes'
+        'sage":"Name is required!"},"response":"Unfortunately an error occ'
+        'ured. You might want to try again. If problem persists, please re'
+        'port this as a bug. We are sorry for inconvenience."}'
+    )
+    try:
+        content = bytes(content, 'utf-8')
+    except:
```

Comment:
`try..except` statement without specifying the exception is really bad antipattern. You could just make `content` a bytestring. Then this whole conversion to bytes would not be needed.